src/App.js

```diff
@@ -1,11 +1,29 @@
-import logo from "./logo.svg";
-import Card from "./Components/Card";
 import "./App.css";
-import Styles from "./Components/Styles.module.css";
-import Courses from "./Components/Courses";
-import CoursesSection from "./Components/CoursesSection";
+import "bootstrap/dist/css/bootstrap.min.css";
+import CoursesSection from "./Components/Courses/CoursesSection";
+import UdemyLogo from "./Components/NavBarComponents/UdemyLogo";
+import NavBar from "./Components/NavBarComponents/NavBar";
+import Button from "react-bootstrap/Button";
+import "bootstrap/dist/css/bootstrap.min.css";
+import Panner from "./Components/ContentHeader.js/Panner";
+import { Route, Routes } from "react-router-dom";
+import HomePage from "./Components/HomePage";
+import CourseHomePage from "./Components/CoursePage/CourseHomePage";
 function App() {
```

Comment:
Many unused imports here , please remove , you can install eslint to lint the unused imports